src/index.js

```diff
@@ -34,6 +34,11 @@ const typeFor = (property: any): string => {
     }
     return `Array<${typeMapping[property.items.type]}>`;
   }
+  else if (property.type === 'string' && 'enum' in property) {
+    return property.enum.reduce((acc: string, e: string) => {
```

Comment:
Thanks for contributing. 
As a minor staff, probably `slice(1)` seems a bit tricky. 
You would be able to replace with below statement.

```js
return property.enum.map(a => `'${a}'`).join(" | ");        
```

Thank you.

Reply:
Thanks for comment.

I fixed this. Please review again.